tests/gradients/parameter_shift/test_parameter_shift.py

```diff
@@ -154,6 +158,160 @@ def test_error_wrong_order(self, order):
             _get_operation_recipe(tape, 0, shifts=None, order=order)
 
 
+class TestMakeZeroRep:
+    """Test that producing a zero-gradient representative with ``_make_zero_rep`` works."""
```

Comment:
I'm a bit surprised this doesn't exist already. Didn't you just extend the function `_make_zero_rep` a little?

Reply:
Yes, but not the entirety of PL is unit-tested :grimacing:

Reply:
Ah, I see. Well, good thing this is covered now 💯